src/bits.rs

```diff
@@ -133,14 +133,14 @@ fn test_push_number() {
     assert_eq!(
         bytes,
         vec![
-            0b010__110__10, // 90
-            0b1__001_1010,  // 154
-            0b1100__1011,   // 203
-            0b0110_1101,    // 109
-            0b01_1001_00,   // 100
-            0b01__111_001,  // 121
-            0b0_1110_001,   // 113
-            0b1__0000000,   // 128
+            0b0101_1010, // 90
+            0b1001_1010, // 154
+            0b1100_1011, // 203
+            0b0110_1101, // 109
+            0b0110_0100, // 100
+            0b0111_1001, // 121
+            0b0111_0001, // 113
+            0b1000_0000, // 128
```

Comment:
revert these and `#[rustfmt::skip]` them, these are intentionally separated like this.

Reply:
I reverted these. `#[rustfmt::skip]` or `#[allow]` are not added.